apps/ios/Shared/Views/Chat/ComposeMessage/ComposeView.swift

```diff
@@ -307,6 +322,10 @@ struct ComposeView: View {
             if let fileName = composeState.voiceMessageRecordingFileName {
                 cancelVoiceMessageRecording(fileName)
             }
+            if composeState.liveMessage != nil {
+                sendMessage()
+                resetLinkPreview()
```

Comment:
sendMessage already calls clearState(), is it not enough?

Reply:
not synchronously.